src/test/scala-2.11/org/apache/james/gatling/smtp/scenari/NoAuthenticationNoEncryptionBigBodyScenario.scala

```diff
@@ -10,22 +10,22 @@ import scala.util.Random
 
 class NoAuthenticationNoEncryptionBigBodyScenario extends Simulation {
 
-  val myRandom = Random.alphanumeric
+  private val myRandom = Random.alphanumeric
 
   def generateMessage() : String =
-    myRandom grouped 200 map (_ append Stream('\r', '\n')) flatMap identity take (1024 * 1024) mkString
+    myRandom.grouped(200).flatMap(_.append(Stream('\r', '\n'))).take(1024 * 1024).mkString
 
-  val users = new UserCreator(BaseJamesWebAdministrationUrl).createUsersWithInboxAndOutbox(UserCount)
+  private val users = new UserCreator(BaseJamesWebAdministrationUrl).createUsersWithInboxAndOutbox(UserCount)
 
-  val scn = scenario("SMTP_No_Authentication_No_Encryption_Big_Body")
+  private val scn = scenario("SMTP_No_Authentication_No_Encryption_Big_Body")
     .feed(UserFeeder.createCompletedUserFeederWithInboxAndOutbox(users))
-    .pause(1 second)
+    .pause(1.second)
     .during(ScenarioDuration) {
       exec(smtp("sendMail")
         .subject("subject")
         .body(generateMessage()))
-        .pause(1 second)
+        .pause(1.second)
```

Comment:
you don't need to do that in every files?

Reply:
I did in only on the file I was modifying to avoid making a huge diff but in general, infix notation should not be used in Scala AFAIK (it's a feature you need to enable explicitly to avoid warning)